node_monitor/node_monitor.py

```diff
@@ -184,7 +185,7 @@ def step(self) -> None:
             # _resync fails, we don't want to analyze or broadcast_alerts.
             self._resync()
             self._analyze()
-            self.broadcast_alerts()
+            asyncio.run(self.broadcast_alerts())
```

Comment:
Why not make the `asyncio.run` call directly in `node_monitor.telegram_bot.TelegramBot`?